plugins/github-actions/src/components/WorkflowRunDetails/useWorkflowRunJobs.ts

```diff
@@ -26,18 +26,41 @@ export const useWorkflowRunJobs = ({
   hostname?: string;
   owner: string;
   repo: string;
-}) => {
+}): AsyncState<Jobs> => {
   const api = useApi(githubActionsApiRef);
   const { id } = useRouteRefParams(buildRouteRef);
-  const jobs = useAsync(async () => {
-    return repo && owner
-      ? api.listJobsForWorkflowRun({
-          hostname,
-          owner,
-          repo,
-          id: parseInt(id, 10),
-        })
-      : Promise.reject(new Error('No repo/owner provided'));
+
+  return useAsync(async () => {
+    if (!repo || !owner) {
+      throw new Error('No repo/owner provided');
+    }
+
+    const jobs = await api.listJobsForWorkflowRun({
+      hostname,
+      owner,
+      repo,
+      id: parseInt(id, 10),
+    });
+
+    return {
+      total_count: jobs.total_count,
+      jobs: jobs.jobs.map<Job>(job => ({
+        html_url: job.html_url || undefined,
```

Comment:
Bit safer to use `??` for this kind of thing, just in case you happen to use this for a numerical value or such

Reply:
Makes good sense. I haven't used that operator knowingly before. Reading up on it it is for sure the better solution. Will update 🙌

Reply:
Code updated to nullish coalescing operator instead 🙌